Flow.Launcher.Infrastructure/UserSettings/PluginSettings.cs

```diff
@@ -30,6 +30,11 @@ public void UpdatePluginSettings(List<PluginMetadata> metadatas)
                         metadata.ActionKeywords = settings.ActionKeywords;
                         metadata.ActionKeyword = settings.ActionKeywords[0];
                     }
+                    else
+                    {
+                        metadata.ActionKeywords = new List<string>();
+                        metadata.ActionKeyword = string.Empty;
```

Comment:
won't this causing "" be used as action keyword?

Reply:
Oh I check the code for adding actionkeyword, only actionkeywords will be access.